pkg/lvgl/contrib/lvgl.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2019 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_lvgl
+ * @{
+ *
+ * @file
+ * @brief       LittlevGL glue code
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @}
+ */
+
+#include "thread.h"
+
+#include "xtimer.h"
+#include "log.h"
+
+#include "lvgl/lvgl.h"
+#include "lv_conf.h"
+#include "lvgl_riot.h"
+#include "disp_dev.h"
+
+#ifndef LVGL_TASK_THREAD_PRIO
+#define LVGL_TASK_THREAD_PRIO   (THREAD_PRIORITY_MAIN + 1)
+#endif
+
+#ifndef LVGL_COLOR_BUF_SIZE
+#define LVGL_COLOR_BUF_SIZE     (LV_HOR_RES_MAX * 5)
+#endif
+
+#ifndef LVGL_ACTIVITY_PERIOD
+#define LVGL_ACTIVITY_PERIOD    (1 * MS_PER_SEC)    /* 1s */
+#endif
+
+#ifndef LVGL_TASK_HANDLER_DELAY
+#define LVGL_TASK_HANDLER_DELAY (5 * US_PER_MS)     /* 5ms */
+#endif
+
+#ifndef LVGL_THREAD_FLAG
+#define LVGL_THREAD_FLAG        (0x4242)
```

Comment:
Thread flags are stored as a bit mask, not as a number. Shouldn't this be something like `(1 << FLAG_NUM)`?